frontend/app/templates/application.hbs

```diff
@@ -8,17 +8,17 @@
           <div class="uk-navbar-dropdown">
             <ul class="uk-nav uk-navbar-dropdown-nav">
               <li class="uk-active">
-                {{#link-to "posts.new" class="menu-posts" }} Create post {{/link-to}}
+                {{#link-to "posts.new" class="menu-create-posts" }} Create post {{/link-to}}
```

Comment:
Do you really need classes here? I think using `ember-test-selectors` would be the better way here..